packages/lit-dev-tools-esm/package.json

```diff
@@ -31,13 +32,27 @@
       ],
       "command": "../../node_modules/.bin/tsc --build --pretty",
       "clean": "if-file-deleted"
+    },
+    "publish-search-index": {
+      "dependencies": [
+        "build"
+      ],
+      "files": [
+        "lib/**"
```

Comment:
not sure if possible to reach out of the project but `../lit-dev-content/_site/searchIndex.json` should also be included, which means `lit-dev-content:build` or whichever command that generates it probably should be a dependency too. if dependency can't be outside the project which i suspect is the case, it might need to be specified in the top level wireit command.

Reply:
muy excelente